hfsutils/configure

```diff
@@ -711,7 +711,7 @@ cross_compiling=$ac_cv_prog_cc_cross
 cat > conftest.$ac_ext <<EOF
 #line 713 "configure"
 #include "confdefs.h"
-main(){return(0);}
+int main(){return(0);}
```

Comment:
This is C code, right? Not C++? If so, specify the function arguments:
```suggestion
int main(void){return(0);}
```

Reply:
Yes, that's right.
However, there also are thousands of this description in other `configure` files.
The purpose of this PR is to make the minimum fix to be able to build on macOS, and I don't intend to fix thousands of things.